region.go

```diff
@@ -0,0 +1,7 @@
+package paperspace
+
+var RegionMap = map[string]int{
+	"East Coast (NY2)": 1,
```

Comment:
Is this not anywhere else?

Reply:
not yet and it can't be retired just yet